log_outgoing_requests/admin.py

```diff
@@ -46,3 +55,20 @@ def query_params(self, obj):
         return obj.query_params
 
     query_params.short_description = _("Query parameters")
+
+    class Media:
+        css = {
+            "all": ("log_outgoing_requests/css/admin.css",),
+        }
+
+
+@admin.register(OutgoingRequestsLogConfig)
+class OutgoingRequestsLogConfigAdmin(SingletonModelAdmin):
+    fields = (
+        "save_to_db",
+        "save_body",
+    )
+    list_display = (
+        "save_to_db",
+        "save_body",
+    )
```

Comment:
`list_display` doesn't mean anything with `SingletonModelAdmin`, there is no list view.